README.md

```diff
@@ -25,6 +25,7 @@ conditions. See the [LICENSE] file for details.
 ## Prerequisites
 
 1. Visual Studio Code with "Remote Development" installed
+2. Using SSH instead of HTTPS to connect to Github ([SSH set-up documentation](https://docs.github.com/en/authentication/connecting-to-github-with-ssh))
```

Comment:
We should mention the GitHub method setting here as well just so people know what is the default.

Reply:
Fair, that would be helpful (will do later today).

I was also debating whether to add that this is a soft rather than a hard prerequisite (since you can always set or mount the git config within the dev container) - but I decided that wasn't critical info and preferred not to add it.

Reply:
Changed, wdyt now?

Reply:
Am I missing something? I don't see any addition of the Visual Studio Code settings.

Reply:
Sorry, what do you mean about the Visual Studio Code settings?

FYI: I realized that the explanation was wrong because I was still getting it (maybe I got the idea that it was due to SSH from another machine where it worked). As my latest change shows, you need a configuration based on whether you have HTTPS or SSH — and I've linked to the walkthrough people should use.